presto-mysql/src/test/java/io/prestosql/plugin/mysql/TestMySqlTypeMapping.java

```diff
@@ -371,6 +376,27 @@ public void testDecimalExceedingPrecisionMaxWithSupportedValues(int typePrecisio
         }
     }
 
```

Comment:
Can you please add tests `io.prestosql.plugin.mysql.BaseMySqlIntegrationSmokeTest#testPredicatePushdown`

Reply:
I've added a test for it, however, I had to create new table because there's no binary data in the test suite - arguably not the prettiest. It should probably be added eventually but that might require a change to all drivers I reckon.